### ISSUE TITLE
XML comments for MD

### ISSUE THREAD

**Issue by ousia** [geany/geany#2179]
Would it be possible that Markdown could have XML comments?

I mean,  this kind of `<!-- comment -->` would make easier working with Markdown sources.

Many thanks for your help.

**Comment by b4n** [comment]
Yes, you can configure that: in [*filetypes.markdown*](https://www.geany.org/manual/#filetype-definition-files), add the corresponding [`comment_open` and `comment_close`](https://www.geany.org/manual/#settings-section) options in the `settings` section.

E.g., *~/.config/geany/filedefs/filetypes.markdown*
```ini
[settings]
comment_open=<!--
comment_close=-->
```

---

We currently don't have any comment marked for Markdown, but I'm not sure there is anything standard in that regard, is there? @codebrainz

**Comment by ousia** [comment]
Many thanks for your help, @b4n.

I’m afraid they only work for adding comments. I mean, toggle commenting/uncommenting only adds more comments 😅.

BTW, these comment marks are standard XML. They should be (at least) valid Markdown.
